meshtastic/interfaces/ble/lifecycle_service.py

```
@@ -0,0 +1,1955 @@
+        Raises
+        ------
+        None
+            Exceptions are suppressed to preserve shutdown progress.
+        """
+        safe_cleanup = BLELifecycleService._resolve_error_handler_hook(
+            iface, "safe_cleanup", "_safe_cleanup"
+        )
+        if safe_cleanup is not None:
+            try:
+                safe_cleanup(cleanup, operation_name)
+                return
```

Ruff (0.15.6)

[warning] Consider moving this statement to an `else` block

(TRY300)